packages/11ty/_includes/components/figure/choices.js

```diff
@@ -0,0 +1,49 @@
+const { globalVault } = require('@iiif/vault')
+const vault = globalVault()
+
+/**
+ * Very simplified method to get choices - expects a valid manifest where choices all have identifiers
+ * @todo replace with vault helper
+ */
+const getChoices = (annotations=[]) => {
+  return annotations.flatMap(({ id, type }) => {
+    const annotation = vault.get(id)
+    if (annotation.motivation.includes('painting')) {
+      const bodies = vault.get(annotation.body)
+      for (const body of bodies) {
+        const { items, type } = body
+        if (type === 'Choice') {
+          return items.map(({ id }) => vault.get(id))
+        }
+      }
+    }
+  }).filter(item => item)
```

Comment:
`flatMap` should make using `filter` to remove `null` items redundant,
```suggestion
  return annotations.flatMap(({ id, type }) => {
    const annotation = vault.get(id)
    if (annotation.motivation.includes('painting')) {
      const bodies = vault.get(annotation.body)
      for (const body of bodies) {
        const { items, type } = body
        return (type === 'Choice') ? items.map(({ id }) => vault.get(id)) : []
      }
    }
  })
```
alternately, use `map().filter()`

Reply:
`flatMap` actually doesn't remove null values, and the `flatMap` is necessary because of the way the data is returned from `vault`.

```
[1, 2, 3, [4]].flatMap((item) => null) // [null, null, null, null]
```